storage/database/db_manager.go

```diff
@@ -1485,6 +1495,68 @@ func (dbm *databaseManager) WriteMerkleProof(key, value []byte) {
 	}
 }
 
+// ReadCode retrieves the contract code of the provided code hash.
+func (dbm *databaseManager) ReadCode(hash common.Hash) []byte {
+	// Try with the legacy code scheme first, if not then try with current
+	// scheme. Since most of the code will be found with legacy scheme.
+	//
+	// TODO-Klaytn-Snapsync change the order when we forcibly upgrade the code scheme with snapshot.
+	db := dbm.getDatabase(StateTrieDB)
+	if data, _ := db.Get(hash[:]); len(data) > 0 {
+		return data
```

Comment:
I don't know if this is a good concept of blockchain.
How about calling 'WriteCode' like the migration method?